### ISSUE TITLE
Uncaught TypeError: Invalid Point: (NaN, NaN)

### ISSUE THREAD

**Issue by whtoo** [atom-haskell/ide-haskell#97]
[Enter steps to reproduce below:]

1. ...
2. ...

**Atom Version**: 1.0.7
**System**: Unknown Windows Version
**Thrown From**: [ide-haskell](https://github.com/atom-haskell/ide-haskell) package, v0.7.1

### Stack Trace

Uncaught TypeError: Invalid Point: (NaN, NaN)

```
At C:\Users\Administrator\AppData\Local\atom\app-1.0.7\resources\app.asar\node_modules\text-buffer\lib\point.js:65

TypeError: Invalid Point: (NaN, NaN)
  at Function.module.exports.Point.assertValid (C:\Users\Administrator\AppData\Local\atom\app-1.0.7\resources\app.asar\node_modules\text-buffer\lib\point.js:65:15)
  at TextBuffer.module.exports.TextBuffer.clipPosition (C:\Users\Administrator\AppData\Local\atom\app-1.0.7\resources\app.asar\node_modules\text-buffer\lib\text-buffer.js:970:13)
  at TextBuffer.module.exports.TextBuffer.markPosition (C:\Users\Administrator\AppData\Local\atom\app-1.0.7\resources\app.asar\node_modules\text-buffer\lib\text-buffer.js:687:49)
  at DisplayBuffer.module.exports.DisplayBuffer.markBufferPosition (C:\Users\Administrator\AppData\Local\atom\app-1.0.7\resources\app.asar\src\display-buffer.js:1414:41)
  at TextEditor.module.exports.TextEditor.markBufferPosition (C:\Users\Administrator\AppData\Local\atom\app-1.0.7\resources\app.asar\src\text-editor.js:1284:62)
  at C:\Users\Administrator\.atom\packages\ide-haskell\lib\editor-control.coffee:188:35
  at queueCmd.callback (C:\Users\Administrator\.atom\packages\haskell-ghc-mod\lib\ghc-modi-process.coffee:161:9)
  at C:\Users\Administrator\.atom\packages\haskell-ghc-mod\lib\ghc-modi-process.coffee:96:24
  at Socket.parseData (C:\Users\Administrator\.atom\packages\haskell-ghc-mod\lib\ghc-modi-process-base.coffee:114:9)
  at Socket.g (events.js:257:16)
  at emitOne (events.js:77:13)
  at Socket.emit (events.js:166:7)
  at readableAddChunk (_stream_readable.js:146:16)
  at Socket.Readable.push (_stream_readable.js:109:10)
  at Pipe.onread (net.js:517:20)

```

### Commands

```
```

### Config

```json
{
  "core": {
    "autoHideMenuBar": true
  },
  "ide-haskell": {}
}
```

### Installed Packages

```coffee
# User
autocomplete-haskell, v0.4.5
haskell-ghc-mod, v0.8.12
ide-haskell, v0.7.1
language-haskell, v1.4.1

# Dev
No dev packages
```

**Comment by lierdakil** [comment]
This is probably related to fix for #92, but I need more information to fix this, since I'm not sure I understand how you run into this.

**Comment by whtoo** [comment]
Perhaps,I got some env error.Therefore I close this for now until I found its root.
